jena/src/main/java/org/apache/commons/rdf/jena/impl/JenaDatasetImpl.java

```diff
@@ -149,7 +149,7 @@ public long size() {
     @Override
     public String toString() {
         final StringWriter sw = new StringWriter();
-        RDFDataMgr.write(sw, graph, Lang.NT);
```

Comment:
It would good to rename graph (throughout the class) as `datasetGraph` or some such. It's not a graph.

Reply:
right.  would something like this be ok?  
```java
    private final DatasetGraph dg;

    JenaDatasetImpl(final DatasetGraph datasetGraph, final UUID salt) {
        this.dg = datasetGraph;
        this.salt = salt;
        this.factory = new JenaRDF(salt);
    }
```
so `dg` is the private variable, and `datasetGraph` is only used in the constructor to match InternalJenaFactory.

Reply:
I'd keep both, parameter and attribute, as `datasetGraph`.

Reply:
Attribute already renamed in `master` (see 151a8ea). 

So, please, rebase this PR to the current `HEAD` so we can ship this patch in the imminent `0.5.0` release.